### PR TITLE
add tempest plugins

### PR DESCRIPTION
**Pull request by guangyee**
Add the tempest plugins for services we care about at the moment.

barbican, heat, and monasca are being added in #1611.

### CODE REVIEW

File: chef/cookbooks/tempest/templates/default/tempest.conf.erb [hunk 1]
```diff
@@ -162,6 +162,9 @@ sahara = <%= @enabled_services.include? "data-processing" %>
 trove = <%= @enabled_services.include? "database" %>
 manila = <%= @enabled_services.include? "sharev2" %>
 magnum = <%= @enabled_services.include? "container-infra" %>
+barbican = <%= @enabled_services.include? "key-manager" %>
+designate = <%= @enabled_services.include? "dns" %>
```
Comment (kbaikov) [inline chef/cookbooks/tempest/templates/default/tempest.conf.erb]: Do we have designate in crowbar?

Comment (guangyee) [reply chef/cookbooks/tempest/templates/default/tempest.conf.erb]: Good question! I can't seem to find it. I'll remove designate-tempest-plugin from the list.

**Comment by guangyee** [comment]
@dirkmueller thanks for the information. Looks like #1611 only has barbican-tempest-plugin change so far. Please advice on the path forward. We can either

1. abandon this one and merge the changes into #1611, or
2. remove the barbican change from this one

File: chef/cookbooks/tempest/templates/default/tempest.conf.erb (hunk 1 above)
Comment (dirkmueller) [reply chef/cookbooks/tempest/templates/default/tempest.conf.erb]: we have an open PR to add designate, so I'd be okay with keeping it in.

**Comment by dirkmueller** [comment]
@guangyee are we looking at the same thing? its adding heat, barbican and monasca. the other tempest plugins weren't added to the product back then (might have changed meanwhile).

File: chef/cookbooks/tempest/templates/default/tempest.conf.erb (hunk 1 above)
Comment (guangyee) [reply chef/cookbooks/tempest/templates/default/tempest.conf.erb]: @dirkmueller done. I've added back designate.

**Pull request merged**